lib/LeagueUtil.js

```diff
@@ -0,0 +1,294 @@
+const deepmerge = require('deepmerge');
+const fs = require('fs');
+const path = require('path');
+
+const ParameterError = require('./errors/ParameterError');
+
+const RateLimiter = require('./RateLimiter');
+
+class LeagueUtil {
+	// TODO: maybe split into more specific classes instead of "Util" class where everything gets dumped into
```

Comment:
Any suggestions depending structure?
Maybe just splitting it into different thematic Util classes would provide best discoverability?